Makefile

```diff
@@ -112,6 +112,18 @@ pause-container-release: pause-container
 	mkdir -p "$(shell pwd)/out"
 	@docker save ${PAUSE_CONTAINER_IMAGE}:${PAUSE_CONTAINER_TAG} > "$(shell pwd)/out/${PAUSE_CONTAINER_TARBALL}"
 
+# Variable to determine branch/tag of amazon-ecs-cni-plugins
+revision=master
```

Comment:
Since, we are working under `aws/amazon-ecs-agent` and are building the `aws/amazon-ecs-cni-plugins`, I think the `get-amazon-ecs` prefix for the target is implicitly understood. I don't see a benefit in typing long Makefile targets with longer variable overrides.

There are no major benefits to making this variable target specific expect the fact that it doesn't apply to any of the other targets.